### PR TITLE
[Docs] Update `getting_started.md` and `install.md` and rewrite `finetune.md`

### PR DESCRIPTION
**Pull request by mzr1996**
## Motivation

There are some problems with our docs. Here I revised the `getting_started.md` and `install.md` and almost rewrote the `finetune.md`.

## Modification

1. As the title.
2. Add `model_zoo.md` and `CONTRIBUTING.md` to our docs.
3. Disable the auto-wrap of code blocks. It may cause some code ugly.

## BC-breaking (Optional)

No

## Use cases (Optional)

No

## Checklist

**Before PR**:

- [x] Pre-commit or other linting tools are used to fix the potential lint issues.
- [x] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [x] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Review by Ezra-Yu — COMMENTED** [review]
we can add  cell that tell the users to frozen some layers in backbone when fine-tune.

File: docs/tutorials/finetune.md [hunk 1]
```diff
@@ -1,94 +1,209 @@
...
 ```python
 _base_ = [
     '../_base_/models/resnet50.py',
-    '../_base_/datasets/cifar10.py', '../_base_/default_runtime.py'
+    '../_base_/datasets/cifar10_bs16.py', '../_base_/default_runtime.py'
 ]
 ```
 
-Besides, users can also choose to write the whole contents rather than use inheritance, e.g. `configs/mnist/lenet5.py`.
+Besides, you can also choose to write the whole contents rather than use inheritance, like `configs/mnist/lenet5.py`.
```
Comment (Ezra-Yu) [inline docs/tutorials/finetune.md:38]: it's better to add link to `configs/mnist/lenet5.py`.

File: docs_zh-CN/tutorials/finetune.md [hunk 2]
```diff
@@ -22,71 +31,166 @@ _base_ = [
...
+```{note}
+这里我们只需要设定我们想要修改的部分配置，其他配置将会自动从我们的父配置文件中获取。
 ```
 
 ## 修改数据集
 
-用户可能还需要准备数据集并编写有关数据集的配置。我们目前支持 MNIST，CIFAR 和 ImageNet 数据集。为了在 CIFAR10 数据集上进行微调，考虑到其原始输入大小为 32，而在 ImageNet 上预训练模型的输入大小为 224，因此我们应将其大小调整为 224。
+当针对一个新的数据集进行微调时，我们通常都需要修改一些数据集相关的配置。比如这
+里，我们就需要把 CIFAR-10 数据集中的图像大小从 32 缩放到 224 来配合 ImageNet 上
+预训练模型的输入。这一需要可以通过修改数据集的预处理流水线（pipeline）来实现。
```
Comment (Ezra-Yu) [inline docs_zh-CN/tutorials/finetune.md:69]: a link to [build owner pipeline](https://mmclassification--466.org.readthedocs.build/en/466/tutorials/data_pipeline.html) could be add here.

File: docs/tutorials/finetune.md (hunk 1 above)
Comment (mzr1996) [reply docs/tutorials/finetune.md:38]: Done.

File: docs_zh-CN/tutorials/finetune.md (hunk 2 above)
Comment (mzr1996) [reply docs_zh-CN/tutorials/finetune.md:94]: I think "custom data pipeline" is not relative to fine-tune. It's better to be mentioned at a start tutorial.

**Comment by mzr1996** [comment]
> we can add cell that tell the users to frozen some layers in backbone when fine-tune.

Done.

**Review by Ezra-Yu — APPROVED** [review]
LGTM.

**Pull request merged**